src/api/app/services/rpmlint_log_parser.rb

```diff
@@ -2,7 +2,7 @@ class RpmlintLogParser
   attr_reader :errors, :badness, :warnings, :info
 
   def initialize(content: '')
```

Comment:
Might be good to make this a requirement here since it doesn't really help much if there's a default that doesn't get applied
```suggestion
  def initialize(content)
```

Reply:
Done. It has been updated